### PR TITLE
Last bulk of special week changes

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? (Required)

Align: 

https://github.com/pingcap/docs-cn/pull/3170/, https://github.com/pingcap/docs-cn/pull/3258, https://github.com/pingcap/docs-cn/pull/3248, https://github.com/pingcap/docs-cn/pull/3149, https://github.com/pingcap/docs-cn/pull/3141/files#diff-9ed840ac86ca303f42c9a0609c6ed526R39

Make information-schema.md -> /information-schema/information-schema.md and fix dead links

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 

- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [x] Change aliases
- [x] Have version specific changes 
- [x] Might cause conflicts

### CODE REVIEW

File: sql-logical-optimization.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+This chapter explains some key logic rewrites to help you understand how TiDB generates the final query plan. For example, when you execute the `select * from t where t.a in (select * from t1 where t1.b=t.b)` query in TiDB, you will find that the `IN` sub-query `t.a in (select t1.a from t1 where t1.b=t.b` does not exist because TiDB has made some rewrites here.
+
+This chapter introduces the following key rewrites:
+
+- [Subquery Related Optimizations](/subquery-optimization.md)
+- [Column Pruning](/column-pruning.md)
+- [Decorrelation of Correlated Subquery](/correlated-subquery-optimization.md)
+- [Eliminate Max/Min](/max-min-eliminate.md)
+- [Predicates Push Down](/predicates-push-down.md)
+- [Partition Pruning](/partition-pruning.md)
```
Comment (yikeke) [inline sql-logical-optimization.md:16]: Waiting for https://github.com/pingcap/docs/pull/3386

File: credits.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
+---
+title: Credits
+summary: Learn about the list and roles of contributors to the TiDB community.
+---
+
+# Credits
+
+The TiDB developer community uses SIG (Special Interest Group) as a unit to manage and organize developers. Each module has its own SIG which is responsible for new feature development, performance optimization, stability guarantee, etc. If you also want to become a TiDB developer, it is undoubtedly the best to join the interested [SIG](https://github.com/pingcap/community/tree/master/special-interest-groups) and discuss face-to-face with other senior developers. As of TiDB 4.0 GA, the following is the list of contributors to the TiDB community and their corresponding roles:
```
Comment (lilin90) [inline credits.md:8]:
```suggestion
The TiDB developer community uses SIG (Special Interest Group) as a unit to manage and organize developers. Each module has its own SIG which is responsible for new feature development, performance optimization, stability guarantee, etc. If you also want to become a TiDB developer, it is undoubtedly the best to join the interested [SIG](https://github.com/pingcap/community/tree/master/special-interest-groups) and discuss directly with other senior developers. As of TiDB 4.0 GA, the following is the list of contributors to the TiDB community and their corresponding roles:
```

File: sql-tuning-overview.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: SQL Tuning Overview
+---
+
+# SQL Tuning Overview
+
+The previous "Troubleshooting" chapter describes some ways to locate some queries that affect the cluster, and that if some queries' execution time does not meet expectations, you need to analyze the execution result to find the cause. In this chapter, the following sections introduce how to tune a specific query:
```
Comment (lilin90) [inline sql-tuning-overview.md:7]:
```suggestion
The previous "Troubleshoot" chapter describes some ways to locate some queries that affect the cluster, and that if some queries' execution time does not meet expectations, you need to analyze the execution result to find the cause. In this chapter, the following sections introduce how to tune a specific query:
```

**Comment by yikeke** [comment]
PTAL again @lilin90

**Comment by nullnotnil** [comment]
LGTM

**Pull request merged**